src/style.css

```diff
@@ -0,0 +1,36 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  border-radius: 0.5px;
```

Comment:
why do we need this border-radius?

Reply:
this is Figma mockup style list
which used to stars element